Week3/hw/app3.js

```diff
@@ -0,0 +1,11 @@
+const userDetails = {
+  username: 'aCoolName',
+  age: 25,
+  chosenColor: 'blue',
+  email: 'aCool@email.com',
+};
+
+function userData(userDetails) {
+  return;
```

Comment:
This function only `return`s, but it does not `console.log` the `userDetails`. The assignment is to `console.log` the `userDetails` inside the function. So can you change that?